datafusion/sqllogictest/src/engines/datafusion_engine/normalize.rs

```diff
@@ -239,6 +239,10 @@ pub fn cell_to_string(col: &ArrayRef, row: usize) -> Result<String> {
                 let key = dict.normalized_keys()[row];
                 Ok(cell_to_string(dict.values(), key)?)
             }
+            // only added because of a bug in v 1.0.4 (is) of lexical-write-integer
```

Comment:
Is this the same as 
- https://github.com/Alexhuszagh/rust-lexical/issues/191

If so I think we could remove the workaround now as it has been subsequently fixed

Reply:
Quite possible, I'll test that out. I had forgotten about that bug - the workaround was to fix that annoyance and I moved on, forgetting even to submit a ticket. Doh.